nix/about.tt

```diff
@@ -88,8 +88,8 @@ there after an upgrade.  This means that you can <em>roll back</em> to
 the old version:</p>
```

Comment:
This diff is nothing but:
1. `<span class="muted">$</span> my command` to `<span class="command">my command</span>`
2. updated a `>` to `&gt;`